lib/msf/core/web_services/views/auth/login.erb

```diff
@@ -7,20 +7,56 @@
 </head>
 <body>
 
+<script
+  src="https://code.jquery.com/jquery-3.3.1.min.js"
```

Comment:
If the user is on an isolated network without internet access then the login form will not operate, correct?

Reply:
This is true. The other options are to include jquery in the repo or to do straight Javascript AJAX. This login form is not really a primary feature at this point, so I'm not sure that it's worth it to write our own yet. When we re-visit this and make it a first class feature we should definitely remove the jquery requirement.